src/Middleware/RequestDecompression/src/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,39 @@
+Microsoft.AspNetCore.RequestDecompression.BrotliDecompressionProvider
```

Comment:
Fixer has a bug where it doesn't add the header we need if it creates a new .txt file ☹️ 
```suggestion
﻿#nullable enable
Microsoft.AspNetCore.RequestDecompression.BrotliDecompressionProvider
```

Reply:
Fixed!